api/facadeversions.go

```diff
@@ -35,6 +35,7 @@ var facadeVersions = map[string]int{
 	"Client":                       1,
 	"Cloud":                        2,
 	"Controller":                   5,
+	"CredentialValidator":          1,
```

Comment:
Would this be better named "CredentialWatcher"?
Will the facade itself be used for any validation of credentials?
Maybe the worker will be a credential validator worker, but the facade's name should reflect its functionality.